ember-cli-build.js

```diff
@@ -28,39 +20,50 @@ const {
   backburnerES,
   dagES,
   routeRecognizerES,
-  glimmerTrees,
+  glimmerES,
+  glimmerCompilerES,
   emberVersionES,
   emberLicense,
-  nodeTests,
-  buildEmberEnvFlagsES,
   getPackagesES,
 } = require('./broccoli/packages');
-const BroccoliDebug = require('broccoli-debug');
-const ENV = process.env.EMBER_ENV || 'development';
 
-let debugTree = BroccoliDebug.buildDebugCallback('ember-source:ember-cli-build');
+const { allSupportedBrowsers, modernBrowsers } = require('./config/browserlists');
 
-module.exports = function() {
-  let loader = internalLoader();
-
-  // generate "loose" ES<latest> modules...
-  let dependenciesES = new MergeTrees([
-    backburnerES(),
-    rsvpES(),
-    dagES(),
-    routerES(),
-    routeRecognizerES(),
+const ENV = process.env.EMBER_ENV || 'development';
+const SHOULD_ROLLUP = process.env.SHOULD_ROLLUP !== 'false';
+const SHOULD_MINIFY = Boolean(process.env.SHOULD_MINIFY);
+
+/**
+ * There isn't a way for us to override targets through ember-cli-babel, and we
+ * don't want to introduce that functionality for reasons. This is a quick and
```

Comment:
I'd like to address this over in ember-cli-babel (in a follow up), would you mind making an issue over there about the general use case?